inst/examples/showcase/static/css/styles.css

```diff
@@ -29,6 +20,61 @@ code[class*=language-], pre[class*=language-] {
 .grid > nav {
   max-height: calc(100vh - 16px);
   overflow: auto;
+  border-right: 1px solid #ddd;
+  height: 100vh;
+}
+
+.section-button {
+  all: unset;
+  display: flex;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  font-size: 16px;
+  width: 100%;
+  margin-top: 20px;
+  padding: 0 10px;
+  cursor: pointer;
+}
+
+.section-button:hover, .section-button:active {
+  text-decoration-line: underline;
+}
+
+.section-button::before, .section-button::after {
+  content: "";
+  position: absolute;
+  right: 25px;
+  width: 1px;
+  height: 60%;
+  margin: 2px 0;
+  border-left: 2px solid #aaa;
+  transform: rotate(90deg);
+  transition: transform 0.3s;
+}
+
+.section-button::after {
+  transform: rotate(0deg);
+}
+
+.section-button.expanded::after {
+  transform: rotate(90deg);
+}
+
+.li-button {
+  list-style-type: none;
+  padding: 5px 0;
+}
+
+.li-button > a {
+  color: black;
+  font-weight: 300;
+}
+
+.li-button > a:focus {
+  outline: none;
+  text-decoration-line: underline;
+  font-weight: 400;
 }
 
```

Comment:
The scrollbar is not attached to the edge of the window. There seems to be a gap

![image](https://user-images.githubusercontent.com/211358/220165129-ffa20287-0bcb-4e99-b80a-c4d515e63004.png)


```suggestion
html,
body {
  margin: 0;
}
```

Reply:
Fixed